frontend/src/Components/MessageUI.tsx

```diff
@@ -128,19 +128,21 @@ export function UserMessageUI({
                   className="u-paddingVerticalExtraSmall"
                   additionalStyles={{ minWidth: 150 }}
                 >
-                  <Dropdown.Item
-                    className="u-cursorPointer u-alignItemsCenter"
-                    role="button"
-                    onClick={() => {
-                      handleReportUser(username);
-                      setShowDropdown(false);
-                    }}
-                  >
-                    <Icon name="flag" size="extraSmall" />
-                    <span className="u-marginLeftExtraSmall u-text200 u-textNoWrap">
-                      Báo cáo vi phạm
-                    </span>
-                  </Dropdown.Item>
+                  {role === 0 && [
+                    <Dropdown.Item
+                      className="u-cursorPointer u-alignItemsCenter"
+                      role="button"
+                      onClick={() => {
+                        handleReportUser(username);
+                        setShowDropdown(false);
+                      }}
+                    >
+                      <Icon name="flag" size="extraSmall" />
+                      <span className="u-marginLeftExtraSmall u-text200 u-textNoWrap">
+                        Báo cáo vi phạm
+                      </span>
+                    </Dropdown.Item>,
+                  ]}
```

Comment:
nit: only add [] when there are more than 1 components.
```suggestion
                  {role === 0 && 
                    <Dropdown.Item
                      className="u-cursorPointer u-alignItemsCenter"
                      role="button"
                      onClick={() => {
                        handleReportUser(username);
                        setShowDropdown(false);
                      }}
                    >
                      <Icon name="flag" size="extraSmall" />
                      <span className="u-marginLeftExtraSmall u-text200 u-textNoWrap">
                        Báo cáo vi phạm
                      </span>
                    </Dropdown.Item>,
                  }
```

Reply:
Thanks @vudung45, I'll update it now